Dockerfile

```
@@ -10,11 +10,13 @@ ENV PYTHONPATH /root
 ARG VERSION
 ARG DOCKER_IMAGE
 
+RUN apt-get update && apt-get install build-essential -y
+
 # Pod tasks should be exposed in the default image
 RUN pip install -U flytekit==$VERSION \
-			flytekitplugins-pod==$VERSION \
-			flytekitplugins-deck-standard==$VERSION \
-			flytekitplugins-data-fsspec==$VERSION \
-			scikit-learn
+	flytekitplugins-pod==$VERSION \
+	flytekitplugins-deck-standard==$VERSION \
+	flytekitplugins-data-fsspec==$VERSION \
```

this should be:
```suggestion
	flytekitplugins-data-fsspec[aws]==$VERSION \
	flytekitplugins-data-fsspec[gcp]==$VERSION \
```

This is because the package `s3transfer` is installed as part of `awscli`, but that was removed in https://github.com/flyteorg/flytekit/pull/1256.

I opened https://github.com/flyteorg/flyte/issues/3049 to help track the work to improve the detection of this kind of error.